src/interfaces/coral_web/src/cohere-client/client.ts

```diff
@@ -318,8 +355,33 @@ export class CohereClient {
     return body as Deployment[];
   }
 
+  public async getExperimentalFeatures(): Promise<ExperimentalFeatures> {
+    const response = await this.fetch(`${this.getEndpoint('experimental_features')}/`, {
+      method: 'GET',
+      headers: this.getHeaders(),
+    });
+
+    const body = await response.json();
+
+    if (response.status !== 200) {
+      throw new CohereNetworkError(
+        body?.message || body?.error || 'Something went wrong',
+        response.status
+      );
+    }
+
+    return body as ExperimentalFeatures;
+  }
+
   private getEndpoint(
-    endpoint: 'upload' | 'chat-stream' | 'conversations' | 'tools' | 'deployments'
+    endpoint:
```

Comment:
I feel it makes more sense to change this to `endpoint: string` at this point 🤔 the list is growing + the method is also private anyway. We're just having to type things out twice

Reply:
though it is repetitive it is nice for the times we do reuse endpoints (e.g. `conversations`) but i don't feel too strongly either way, down to update it next time we add an endpoint!